roles/openshift_metrics/vars/openshift-enterprise.yml

```diff
@@ -1,3 +1,3 @@
 ---
 __openshift_metrics_image_prefix: "{{ openshift_hosted_metrics_deployer_prefix | default('registry.access.redhat.com/openshift3/') }}"
-__openshift_metrics_image_version: "{{ openshift_hosted_metrics_deployer_version | default ('3.6.0') }}"
+__openshift_metrics_image_version: "{{ openshift_image_tag | default(openshift_hosted_metrics_deployer_version) | default ('v3.6') }}"
```

Comment:
where is `openshift_image_tag` coming from? And should we consider using that? We can override this value being defaulted with `openshift_metrics_image_version` in the inventory.